UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -138,11 +148,25 @@ public async Task<string> CreateLargeExchangeSet(SalesCatalogueServiceResponseQu
 
             if (aioConfiguration.IsAioEnabled)
             {
+                var largeExchangeSetDataResponse = new LargeExchangeSetDataResponse()
```

Comment:
```suggestion
                var largeExchangeSetDataResponseForAio = new LargeExchangeSetDataResponse()
```
Can we change name as suggested to make variable meaningful?

Reply:
Resolved